src/components/organisms/LocalReplace.tsx

```diff
@@ -0,0 +1,448 @@
+          break
+        case 'e':
+        case 'E':
+          if (event.altKey) {
+            event.preventDefault()
+            event.stopPropagation()
+            toggleCaseSensitiveReplace()
+          }
+          break
+        case 'F3':
+          if (event.shiftKey) {
+            navigateToNext('previous')
+            focusReplaceTextAreaInput(replaceQuery.length)
+          } else {
+            navigateToNext('next')
+            focusReplaceTextAreaInput(replaceQuery.length)
+          }
+          break
+        case 'ArrowDown':
+          if (getNumberOfTextAreaRows <= 1) {
+            event.preventDefault()
+            event.stopPropagation()
+            navigateToNext('next')
+            focusReplaceTextAreaInput(replaceQuery.length)
+          }
+          break
+        case 'ArrowUp':
+          if (getNumberOfTextAreaRows <= 1) {
+            event.preventDefault()
+            event.stopPropagation()
+            navigateToNext('previous')
+            focusReplaceTextAreaInput(replaceQuery.length)
+          }
+          break
+        case 'Escape':
+          handleOnReplaceClose()
+          break
+        case 'f':
+        case 'F':
+          if (event.ctrlKey && onReplaceToggle) {
+            onReplaceToggle(false)
+          }
+          break
+      }
+    },
+    [
+      addNewlineToReplaceValue,
+      focusReplaceTextAreaInput,
+      handleOnReplaceClose,
+      navigateToNext,
+      onFocusSearchInput,
+      onReplaceCurrentItem,
+      onReplaceToggle,
+      replaceQuery.length,
+      getNumberOfTextAreaRows,
+      toggleCaseSensitiveReplace,
+    ]
+  )
+
+  useEffect(() => {
+    if (focusingReplace) {
+      // focusReplaceTextAreaInput(replaceQuery.length)
+      focusReplaceTextAreaInput()
+    }
+  }, [focusReplaceTextAreaInput, focusingReplace, replaceQuery.length])
+
+  return (
+    <LocalReplaceContainer>
+      <SearchResultItem>
+        <LocalSearchInputLeft>
+          <LocalReplaceIcon
+            className={getNumberOfTextAreaRows != 1 ? 'alignStart' : ''}
+          >
+            <Icon path={mdiMagnify} />
+          </LocalReplaceIcon>
+          <textarea
+            onClick={focusReplaceTextAreaInput}
+            rows={getNumberOfTextAreaRows}
+            value={replaceValue}
+            onInput={updateReplaceValue}
+            onChange={updateReplaceValue}
+            // onChange={() => undefined}
+            onKeyDown={handleReplaceInputKeyDown}
+            ref={replaceTextAreaRef}
+          />
+          <SearchOptionsInnerContainer>
+            <LocalSearchStyledButton
+              title={'Enter newline'}
+              className={'button'}
+              onClick={addNewlineToReplaceValue}
+            >
+              <Icon path={mdiSubdirectoryArrowLeft} />
+            </LocalSearchStyledButton>
+            <LocalSearchStyledButton
+              title={
+                'Preserve case Alt+E - Use tab to focus on an option and space to toggle'
```

Comment:
Do you have any idea how to improve this?
Here is an image how it looks in tooltip in Webstorm:
![image](https://user-images.githubusercontent.com/18196945/109067270-95a75e80-76ee-11eb-963c-7a0ffaaac5bb.png)
I would like to have such toolitp, but I think this is for another PR (and when tooltip implementation is merged)
For now it would show everything in one line. 
Also maybe we should add parenthesses around shortcut or somehow style it differently like shown in the above image?

Reply:
Yeah, I think adding parentheses is enough for now. Let's deal with the tooltip after we merge the other pr.

Reply:
Added and updated New Line button to include tooltip with shortcut for adding newline (ctrl+shift+enter)